fabfile.py

```diff
@@ -88,17 +82,17 @@ def get_db_dump(dbname, clean=True):
 
     """
     require('environment')
-    if not files.exists("%(home)s/.pgpass" % env):
-        abort("Please get a copy of .pgpass and put it in your home dir on the server of interest (not your local system)")
+    run('sudo -u pycon /srv/pycon/env/bin/python /srv/pycon/pycon/manage.py sqldsn -q -s pgpass -R default 2>/dev/null > ~/.pgpass')
+    run('chmod 600 ~/.pgpass')
     dump_file = '%(project)s-%(environment)s.sql' % env
     flags = '-Ox'
+    dsn = sudo('/srv/pycon/env/bin/python /srv/pycon/pycon/manage.py sqldsn -q -R default 2>/dev/null', user='pycon').stdout
```

Comment:
no, there are two different forms pulled from the `sqldsn` manage command:

`/srv/pycon/env/bin/python manage.py sqldsn -q 2> /dev/null`
gives
`host='postgresql.psf.io' dbname='pycon-staging' user='pycon-staging' port='5432' sslmode=verify-full hostaddr=192.168.5.11 sslrootcert=/etc/ssl/certs/PSF_CA.pem`

which is required to perform the SSL connection to the psf infrastructure

`/srv/pycon/env/bin/python manage.py sqldsn -q -s pgpass 2> /dev/null`
yields
`postgresql.psf.io:5432:pycon-staging:pycon-staging:<redacted-password>`

which can be used for providing the password without prompt.

these are required since the postgresql infrastructure for psf-salt is dynamic.